NuGet.config

```diff
@@ -6,6 +6,13 @@
   <!-- Only specify feed for Arcade SDK (see https://github.com/Microsoft/msbuild/issues/2982) -->
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
```

Comment:
@mmitche My understanding is that these should be removed, but we are intentionally pulling some 3.0.0 packages that won't restore without them. Did something not get pushed to the right place?

Reply:
I think I saw elsewhere that some packages were uploaded to dotnet3 feed, let me retry without these again.